src/containers/Tabs.tsx

```diff
@@ -56,4 +67,38 @@ const TabsContainer = ({ tabsData, defaultActiveKey }: TabsProps) => {
     </StyledTabs>
   );
 };
+
+export const LinkTabsContainer = ({
```

Comment:
LinkTabsContainer component duplicated, it is possible to use one component - 'TabsContainer'. This situation also with LinkTabsDataProps and LinkTabsProps